edx_exams/apps/api/v1/views.py

```diff
@@ -579,6 +579,29 @@ def post(self, request):
         data = {'exam_attempt_id': exam_attempt_id}
         return Response(data)
 
+    def delete(self, request, attempt_id):
+        """
+        HTTP DELETE handler to delete exam attempt
+        """
+        exam_attempt = get_attempt_by_id(attempt_id)
+        if exam_attempt is None:
+            return Response(
+                status=status.HTTP_400_BAD_REQUEST,
+                data={'detail': f'Attempt with attempt_id={attempt_id} does not exit.'}
```

Comment:
Nit: should be exist not exit.
```suggestion
                data={'detail': f'Attempt with attempt_id={attempt_id} does not exist.'}
```